src/Dialogs/StripeDialog.vala

```diff
@@ -427,5 +437,43 @@ public class AppCenter.Widgets.StripeDialog : Gtk.Dialog {
 
         return data.str;
     }
+
+    private string get_stripe_error (Json.Object error_object) {
```

Comment:
Can we make this static? Doesn't look like it's accessing any members from the instance. Also, this function should return an `unowned string` since `_` function you're using does return an `unowned`: https://valadoc.org/glib-2.0/GLib._.html